packages/react/src/components/Rating/Rating.base.tsx

```diff
@@ -178,7 +183,8 @@ export const RatingBase: React.FunctionComponent<IRatingProps> = React.forwardRe
           {...(readOnly && {
             allowFocusRoot: true,
             disabled: true,
-            'aria-label': ariaLabel,
+            role: 'textbox',
+            'aria-label': readOnlyAriaLabel,
```

Comment:
I wonder if this should fall back to the normal mode aria label if getAriaLabel wasn't defined? And/or add a note on `ariaLabel` in the types file explaining that if the control is read-only, it's recommended to provide `getAriaLabel` since otherwise the current rating value may not be read.